.github/workflows/nightly.yml

```diff
@@ -0,0 +1,36 @@
+---
+# A schedule only runs on branches that match include rules from _both_ `main`
+# and the branch itself.  On `main`, we blanket include all branches that might
+# want to be enabled, then particular branches can override it to exclude
+# themselves by removing themselves from the trigger list.  For example, old
+# stable branches can remove `stable/*` from their copy of this file once they
+# reach their end-of-life.
+name: Nightly Main tests
+on:
+  # remove when proven to work
+  pull_request:
+  schedule:
+    - cron: "20 6 * * *"
+  workflow_dispatch:
+
+jobs:
+  nightly-tests:
+    # Nightly cron job.
+    #
+    # For this to run on a branch, the `schedules` trigger  up at the top of this
+    # file needs to match on _both_ `main` and the branch itself.
+    if: ${{ github.repository_owner == 'Qiskit'}}
+    uses: ./.github/workflows/on-nightly.yml
+  nightly-failure:
+    name: Create comment on failed test run
+    if: ${{ github.repository_owner == 'Qiskit' && failure() }}
+    runs-on: ubuntu-latest
+    needs: [nightly-tests]
+    steps:
+      - uses: peter-evans/create-or-update-comment@v4
```

Comment:
There's not a more official action for this?

Reply:
This is what was being used by other workflows that do the same. I could change it here but the other ones will need to be changed on a separate PR

Reply:
Switched to gh CLI in 0466f92